### ISSUE TITLE
Data information used in cutscenes are hardcoded

### ISSUE THREAD

**Issue by RaptorParkowsky** [colobot/colobot#725]
All basic informations about textures of planets, sky, placement and camera moves in GOLD's cutscenes are currently hardcoded.

**Comment by krzys-h** [comment]
I somehow missed your commit colobot/colobot-data@960980525841b75effcfeb7a5b5eb6bdb7d5b735, now I know what you actually meant with this issue :laughing: 
The planet textures are not hardcoded, they are set in https://github.com/colobot/colobot-data/blob/dev/levels/missions/chapter002/level001/scene.txt#L10-L12

**Comment by RaptorParkowsky** [comment]
Yeah, I totally missed that, sorry. But still there's somehardcoded stuff like camera movement, etc.

And I can't find any parameter that would be responsible for background/sky texture.

**Comment by krzys-h** [comment]
Background is hardcoded, camera too.
